DevXonnect/Xonnect-Currency-Converter
# UI layout for Currency Converter App

**What does this PR do?**
UI layout for Currency Converter Application

**Description of Task to be completed?**
A single 1-columned layout page with a search input field

**How should this be manually tested?**
1. git clone https://github.com/DevXonnect/Xonnect-Currency-Converter
2. cd Xonnect-Currency-Converter
3. npm install
4. npm start
5. The application should be run on the browser on port 3000

**Screenshots**
![selection_071](https://user-images.githubusercontent.com/29041253/49931795-53f4b980-fed0-11e8-89f0-c34916575914.png)
